sql/moz-fx-data-shared-prod/telemetry_dev_cycle_derived/glean_metrics_stats_v1/query.sql

```diff
@@ -34,12 +35,15 @@ final AS (
     last_date,
     expires,
     CASE
+      WHEN NOT in_source
+        AND expiry_date IS NULL
+        THEN last_date
+      WHEN NOT in_source
+        THEN IF(expiry_date < last_date, expiry_date, last_date)
       WHEN expiry_date IS NULL
-        AND DATE_DIFF(CURRENT_DATE(), last_date, day) < 2
+        OR expiry_date > CURRENT_DATE()
```

Comment:
Is this a persisted table?
If so, is it incrementally updated or is everything overwritten everday and thus this check actually runs again every day?

Reply:
everything is run every day ... it's tiny